### PR TITLE
Selectable Text

### PR DESCRIPTION
**Pull request by VB10**
Hello dear
I need to select in text value so add selectable options core widget.

### CODE REVIEW

**Comment by daohoangson** [comment]
What did you use to format the code? Please refrain from reformatting unrelated files from your PR. It's a bit hard to review at this state.

**Comment by VB10** [comment]
may formatter did not this, my vs code conifg  created this changes (number of line dart), i will fix that to send again

**Comment by daohoangson** [comment]
That would be great. Thank you ❤️

**Comment by VB10** [comment]
i fixed formatting error thanks you everyting <3 @daohoangson

**Comment by dan-leech** [comment]
I checked this on iOS but I cannot select a text.
How does it work? I've not found any example also.

I edited demo_app `hello_world_core.dart`, add isSelectableText: true but it seems it does nothing.

**Comment by VB10** [comment]
@dan-leech you right, I'll add an example at the demo folder

File: packages/core/lib/src/core_widget_factory.dart [hunk 1]
```diff
@@ -194,6 +195,16 @@ class WidgetFactory {
         maxLines: tsh?.maxLines == -1 ? null : tsh?.maxLines,
       );
 
+  /// Builds [RichText].
+  Widget buildSelectableText(
```
Comment (daohoangson) [inline packages/core/lib/src/core_widget_factory.dart:199]: This method is not being used anywhere.

**Comment by VB10** [comment]
@dan-leech  you can look this image, just define selectableTrue and you did it.
image

**Review by daohoangson — CHANGES_REQUESTED** [review]
I have cleaned up the PR to minimize the number of changes so it can be reviewed efficiently. At the current state, setting `isSelectableText: true` doesn't do anything. So maybe you need to double check the commits.

PS: It's unclear which IDE you are using but if it auto-formats again, you can execute `flutter format .` from the command line to reformat the files.

**Comment by daohoangson** [comment]
This doesn't seem to work with inline images? Or SUP/SUB tags?

**Comment by VB10** [comment]
yeah, this needs some improvement but the structure has more complexity, I'll develop well solution after to create pr again.

**Comment by daohoangson** [comment]
Understood. Let's keep poking this thing.

**Pull request closed without merging**